source/_integrations/squeezebox.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 title: Squeezebox (Lyrion Music Server)
-description: Instructions on how to integrate a Squeezebox player into Home Assistant.
+description: Instructions on how to integrate a Lyrion Music Server(LMS) and squeezebox players into Home Assistant.
```

Comment:
```suggestion
description: Instructions on how to integrate Squeezebox players and a Lyrion Music Server (LMS)  into Home Assistant.
```
As the integration is called Squeezebox, Squeezebox should probably be mentioned before the music server.